packages/records/lib/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './db/migrate.js';
+export * as Records from './models/records.js';
```

Comment:
I think it's common to only consider PascalCase for class, wdyt?

Reply:
I personally don't like wildcard import name with lowercase name. Ok it is not a class but it is basically a namespace. 
If the convention in js land is to use lowercase I will change it